src/fluree/db/json_ld/policy.cljc

```diff
@@ -488,10 +489,12 @@
          (throw e))))))
 
 
-(defn policy-opts
-  [opts]
-  (-> (select-keys opts [:did :role :credential])
-      not-empty))
+(defn policy-identity
+  [{:keys [context] :as identity-map}]
+  (when-let [{:keys [role] :as identity} (-> (select-keys identity-map [:did :role :credential])
+                                             not-empty)]
+    (cond-> identity
+      (and role context) (update :role json-ld/expand-iri (json-ld/parse-context context)))))
```

Comment:
Another take it or leave it thing (and this is even more personal preference than style): I'm not a huge fan of using `cond->` w/ only one condition. I think plain ol' `if` `else` is much more readable for these scenarios, especially for outside contributors.